tests/fresh_versions_test.ts

```diff
@@ -0,0 +1,26 @@
+import { resolve } from "jsr:@std/path";
+import { assertEquals } from "jsr:@std/assert";
+
+Deno.test("Fresh version consistency", async () => {
+  const config = JSON.parse(await Deno.readTextFile(resolve("deno.json")));
+  // get the expected version from the deno.json file
+  const expectedVersion = config.version;
+  if (!expectedVersion) {
+    throw new Error("No version field found in deno.json");
+  }
+
+  // get the version from the fresh import
+  const freshImport = config.imports?.fresh;
```

Comment:
```suggestion
  const freshImport = config.imports.fresh;
```
Ditto, safe to assume `imports` field will always be defined.